Task_Position_the_Cubes/index.html

```diff
@@ -5,7 +5,18 @@
     <link rel="stylesheet" href="index.css" />
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for POSITION THE CUBES TASK-->
+  <body>
+    <div class="outer-box">
+      <div class="inner-box">
+      <div class="inner-box1">
+      <div class="inner-box2">
+      <div class="inner-box3">
+      <div class="small-square">
+      </div>
+      </div>
```

Comment:
The closing HTML tag should include the DOCTYPE declaration and not include the '</html>' tag.